macOS/DuckDuckGo/UnifiedFeedbackForm/UnifiedFeedbackFormViewModel.swift

```diff
@@ -130,6 +130,15 @@ final class UnifiedFeedbackFormViewModel: ObservableObject {
     let source: UnifiedFeedbackSource
     private(set) var availableCategories: [UnifiedFeedbackCategory] = [.selectFeature, .subscription]
 
+    var availableSubscriptionSubcategories: [SubscriptionFeedbackSubcategory] {
+        var subcategories: [SubscriptionFeedbackSubcategory] = [.selectSubcategory, .otp]
+        if featureFlagger.isFeatureOn(.allowProTierPurchase) {
+            subcategories.append(.unableToAccessFeatures)
+        }
+        subcategories.append(.somethingElse)
+        return subcategories
```

Comment:
I would avoid building the array of subcategories by starting with a subset like in a line 134. That way in case `SubscriptionFeedbackSubcategory` changes in the future it will be required to keep this place in code in sync as well.

I'd suggest starting with `SubscriptionFeedbackSubcategory.allCases` but in the case of feature flag being off I would remove the `.unableToAccessFeatures` option.